cpu/native/socket_zep/socket_zep.c

```diff
@@ -383,16 +383,16 @@ void socket_zep_setup(socket_zep_t *dev, const socket_zep_params_t *params)
     assert(ss_len >= IEEE802154_LONG_ADDRESS_LEN);
 
     /* generate hardware address from socket address and port info */
-    dev->netdev.long_addr[1] = 'Z';     /* The "OUI" */
-    dev->netdev.long_addr[2] = 'E';
-    dev->netdev.long_addr[3] = 'P';
+    dev->netdev.long_addr.uint8[1] = 'Z';     /* The "OUI" */
+    dev->netdev.long_addr.uint8[2] = 'E';
+    dev->netdev.long_addr.uint8[3] = 'P';
     for (unsigned i = 0; i < ss_len; i++) { /* generate NIC from local source */
         unsigned addr_idx = (i % (IEEE802154_LONG_ADDRESS_LEN / 2)) +
                             (IEEE802154_LONG_ADDRESS_LEN / 2);
-        dev->netdev.long_addr[addr_idx] ^= ss_array[i];
+        dev->netdev.long_addr.uint8[addr_idx] ^= ss_array[i];
     }
-    dev->netdev.short_addr[0] = dev->netdev.long_addr[6];
-    dev->netdev.short_addr[1] = dev->netdev.long_addr[7];
+    dev->netdev.short_addr.u8[0] = dev->netdev.long_addr.uint8[6];
+    dev->netdev.short_addr.u8[1] = dev->netdev.long_addr.uint8[7];
```

Comment:
But this is supposed to be an improvement?

Reply:
This argument applies to the use of `eui64_t` in general, not to using it consistently.

But in fact the use of `eui64_t` will allow making this a bit easier, I will add a fixup to do this.